multiraft/multiraft.go

```diff
@@ -921,6 +953,7 @@ func (s *state) processCommittedEntry(groupID proto.RangeID, g *group, entry raf
 					// redundant since most pending proposals won't benefit from this but
 					// config changes should be rare enough (and the size of the pending queue
 					// small enough) that it doesn't really matter.
```

Comment:
This comment is no longer accurate; replace it with "Re-submit all pending proposals that were held while the config change was pending".